wrapper

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
-
-version="0.41.0"
+TMPDIR=/a/tmp export TMPDIR
```

Comment:
I think this is an error.

Reply:
say more? Is it the `FOO=bar export BAR` pattern (which is unusual, but valid) or something else?
If it's the 0.41.0, then yes, that doesn't reflect deployment reality and I'll adjust that.

Reply:
Actually, the PR already makes that change to 0.42